app/uk/gov/hmrc/gform/views/form/snippets/lookup_autosuggest.scala.html

```diff
@@ -71,8 +71,8 @@
       data-displayWidth="@displayWidth">
       </div>
       <noscript>
-          @selectItems.zipWithIndex.map { case (items, index) =>
-            @govukSelect(Select(id = s"${fcId}", name = s"${fcId}_$index",  items = items))
+          @selectItems.map { case items =>
+            @govukSelect(Select(id = fcId, name = fcId,  items = items))
```

Comment:
Are we going in circles here? 

Why was this done in the first place: https://github.com/hmrc/gform-frontend/pull/1938/files#diff-77ad5191f03d6c49b2fe1034cb004f2e57d5b42c4b991b1d0f89364ecd1f1e86L72-R73 ?

Reply:
This accessibility report cannot be accessed. I don't remember the reason for this change. I will run the accessibility report but the ID must be the same as the name

Reply:
We run our accessibility report on this branch. It passed all tests.